caps/templates/includes/council-powers.html

```diff
@@ -0,0 +1,135 @@
+                <li>incentivising Electric Vehicle use by assigning street space to EV charging</li>
+                <li>disincentivising the use of traditional cars through congestion charging or the reduction of parking space</li>
+            </ul>
+        </div>
+    </details>
+{% endif %}
+
+{% if council.authority_type == 'UA' or council.authority_type == 'MD' or council.authority_type == 'CTY' %}
+    <details class="details details-accordion">
+        <summary class="mb-2">Passenger transport</summary>
+        <div class="council-powers">
+            <p>Climate actions might include:</p>
+            <ul>
+                <li>encouraging bus and private hire operators to use low-emission vehicles, through licensing requirements or Clean Air Zones</li>
+                <li>incentivising bus use by improving routes, timetables, and ticket prices through ‘Enhanced Partnerships’ with operators</li>
+                <li>proposing to central government the development of light rail / tram networks that integrate, rather than compete, with other modes of transport in the area</li>
+            </ul>
+        </div>
+    </details>
+{% endif %}
+
+{% if council.authority_type != 'COMB' and council.authority_type != 'NMD' %}
+    <details class="details details-accordion">
+        <summary class="mb-2">Schools and libraries</summary>
+        <div class="council-powers">
+            <p>Climate actions might include:</p>
+            <ul>
+                <li>reducing the carbon footprint of civic buildings through better insulation and renewable energy use</li>
+                <li>incentivising ‘Active Travel’ or public transport use among employees</li>
+            </ul>
+        </div>
+    </details>
+{% endif %}
+
+{% if council.authority_type != 'COMB' and council.authority_type != 'CTY' %}
+    <details class="details details-accordion">
+        <summary class="mb-2">Environmental health</summary>
+        <div class="council-powers">
+            <p>Climate actions might include:</p>
+            <ul>
+                <li>reducing industrial emissions through air quality enforcement</li>
+                <li>bringing up insulation and energy efficiency standards through enforcement of the Housing Health and Safety Rating System (HHSRS) and Decent Homes Standard 2000. for cold and damp conditions in private rental and social housing</li>
+            </ul>
+        </div>
+    </details>
+{% endif %}
+
+{% if council.authority_type != 'COMB' %}
+    <details class="details details-accordion">
+      {% if council.authority_type != 'CTY' and council.authority_type != 'NMD' %}
+        <summary class="mb-2">Waste collection and disposal</summary>
+      {% elif council.authority_type != 'CTY' %}
+        <summary class="mb-2">Waste collection</summary>
+      {% elif council.authority_type != 'NMD' %}
+        <summary class="mb-2">Waste disposal</summary>
+      {% endif %}
```

Comment:
This set of conditions is a bit confusing. If I'm correct this might be better expressed as
if not CTY and not NMD
    Waste collection and disposal
elif CTY
    Waste disposal
else
    Waste collection

Reply:
Done! (Except now it’s done in the model instead of the template, yay!)